frontend/src/components/commons/Tab.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+const styles = {
+  selected: {
+    color: "black"
+  },
+  unselected: {
+    color: "blue"
+  }
+};
+
+class Tab extends Component {
+  static propTypes = {
+    tabName: PropTypes.string.isRequired,
+    selected: PropTypes.bool.isRequired
+  };
+
+  render() {
+    return (
+      <span>
+        {this.props.selected === false && (
+          <button className={this.props.tabName} style={styles.unselected}>
+            {this.props.tabName}
+          </button>
+        )}
+        {this.props.selected === true && (
+          <button className={this.props.tabName} style={styles.selected}>
+            {this.props.tabName}
+          </button>
+        )}
+      </span>
+    );
+    //TODO can we pass a function as a prop??
```

Comment:
Yes we can! We did it in the Translation PR, but can we remove this comment before merging?